.github/workflows/release.yml

```diff
@@ -0,0 +1,123 @@
+name: Release
+
+env:
+  GO111MODULE: "on"
+  GOPATH: "/home/runner/work/armada/armada/go"
+  GOCACHE: "/home/runner/.cache/go-build"
+
+on: 
+  push:
+    tags: 
+      - 'v*'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-20.04
+    env:
+      GO111MODULE: "on"
+    steps:
+      - name: Checkout code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          path: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+      
+      - name: Build armadactl release artifacts
+        run: |
+          TAG=${GITHUB_REF##*/}
+          make build-armadactl-release RELEASE_VERSION=${TAG}
+        working-directory: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+
+      - name: Archive production artifacts  
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: |
+            ${{ env.GOPATH }}/src/github.com/G-Research/armada/dist/*
+
+      #### Publish release
+      - name: Upload artifacts to Github release #TODO change RELEASE_TOKEN
+        uses: ncipollo/release-action@v1.8.6
```

Comment:
I assume you've tested what this outputs and gives up something equivalent to what we already publish to our releases.
 - If so, using a library is even better than our custom script :)

Reply:
confirming 😄 + commits are hyperlinks 🚀 
[New look](https://github.com/ljubon/armada/releases/tag/v0.3.97) vs [Old look](https://github.com/G-Research/armada/releases/tag/v0.2.3)